packages/dds/tree/api-report/tree.api.md

```diff
@@ -1662,6 +1662,16 @@ export interface SharedTreeOptions extends Partial<ICodecOptions> {
     treeEncodeType?: TreeCompressionStrategy;
 }
 
+// @public
+export interface SimpleTreeApi {
```

Comment:
the concept of "simple" tree (and the implied alternative) is a concept we haven't (and may never) expose to users. makes me wonder if something like "TreeContentApi" or similar is better.

Reply:
Good point. I'll change it to `TreeNodeApi`, since these APIs are specifically about getting information from individual nodes. In contrast, the transaction functions in `TreeApi` don't have much to do with the nodes themselves and are more just alongside them for convenience.